python/ray/serve/tests/test_telemetry.py

```diff
@@ -110,6 +110,45 @@ def start_telemetry_app():
     return storage
 
 
+# NOTE(zcin): This test needs to be run first because calls to serve.status()
+# in this file will affect the telemetry collected.
```

Comment:
this is not acceptable, it will lead to subtle bugs in the future. please try to re-write the test in a way that it does not produce side effects